notebooks/hydrogen_market.ipynb

```diff
@@ -0,0 +1,269 @@
+    "config_path = PARENT + f\"/config.yaml\"\n",
+    "config = yaml.safe_load(open(config_path))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Hydrogen Market"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 64,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Hydrogen Market is: 12.82 €\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Market\n",
+    "h2_market = (statistics.loc[\"Link\", \"H2 Electrolysis\"][\"Market Value\"]).round(2)\n",
+    "print(f\"Hydrogen Market is: {h2_market} €\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Hydrogen Costs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Capital Expenditure is: 210.44656 Mio. €\n",
+      "Operational Expenditure is: 0.21842496 Mio. €\n",
+      "Total Hydrogen Production Cost: 210.66498496 Mio. €\n",
+      "Hydrogen Production Cost per Unit: 12.819730658113464 Mio. € per TWh\n"
+     ]
+    }
+   ],
+   "source": [
+    "h2_capex = (\n",
+    "        statistics.loc[\"Link\", \"H2 Electrolysis\"]\n",
+    "        .loc[\"Capital Expenditure\"]\n",
+    "        .round(2)\n",
+    "        / 1e6\n",
+    ")  # in Mio. €\n",
+    "print(f\"Capital Expenditure is: {h2_capex} Mio. €\")\n",
+    "\n",
+    "h2_opex = (\n",
+    "    statistics.loc[\"Link\", \"H2 Electrolysis\"]\n",
+    "    .loc[\"Operational Expenditure\"]\n",
+    "    .round(2)\n",
+    "    / 1e6\n",
+    ")  # in Mio. €\n",
+    "print(f\"Operational Expenditure is: {h2_opex} Mio. €\")\n",
+    "\n",
+    "hydrogen_production = (\n",
+    "    statistics.loc[\"Link\", \"H2 Electrolysis\"]\n",
+    "    .loc[\"Supply\"]\n",
+    "    .sum() \n",
+    "    / 1e6\n",
+    ")  # in TWh\n",
+    "\n",
+    "total_hydrogen_production_cost = h2_capex + h2_opex\n",
+    "\n",
+    "hydrogen_cost_per_unit = total_hydrogen_production_cost / hydrogen_production\n",
+    "\n",
+    "# Print Results\n",
+    "print(f\"Total Hydrogen Production Cost: {total_hydrogen_production_cost} Mio. €\")\n",
+    "print(f\"Hydrogen Production Cost per Unit: {hydrogen_cost_per_unit} Mio. € per TWh\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Hydrogen Exports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 65,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Hydrogen Export is: 333662848.5062631 MWh\n"
+     ]
+    }
+   ],
+   "source": [
+    "H2_export_MWh = (\n",
+    "            n.stores[\n",
+    "                (n.stores.carrier == \"H2\") & (n.stores.bus == \"H2 export bus\")\n",
+    "            ].e_nom_opt.sum()\n",
+    "        )  # in GWh\n",
```

Comment:
This is the hydrogen export store size, but not the exported hydrogen volume. You can use 
`n.statistics.energy_balance(comps=["Load"])` and in there you should be able to find a load called "H2 export load". This should give you the desired export volume.

Reply:
@huyhoang-mike looks good, the resulting value is correct. One addition regarding the `H2_export_revenue = H2_export_TWh*h2_market`: Technically more precise would be to multiply quantity and price on an hourly basis (and not yearly as you are doing). Reason for this is that the hydrogen price is hourly dependent. However, you could also just add a printed warning, stated that the value is calculated on a yearly basis :)